docs/security-assessment.md

```diff
@@ -0,0 +1,99 @@
+# Security Assessment PURIS (incl. Frontend, Backend Services, IAM and other infrastructure)
+
+| Contact                  | Details                                                                           |
+| ------------------------- | ---------------------------------------------------------------------------------------------- |
+| Contact for product       | [@tom-rm-meyer-ISST](https://github.com/tom-rm-meyer-ISST) |
+| Security responsible      | [@SSIRKC](https://github.com/SSIRKC) <br> [@szymonkowalczykzf](https://github.com/szymonkowalczykzf) |
+| Version number of product | 23.12                                                                                          |
```

Comment:
Security Assessment has been conducted for Release 24.03. Thus it is correct, isn't it? Just want to confirm

Reply:
Hi Tom, you need to schedule a new assessment for 24.03. This is only the migration to GitHub :)

Reply:
Ah ok, We didn't participate in R23.12 and therefore just wanted to be fast enough for R24.03. Also when talking about prioritization things have been postponed for R24.05.

But I can open a ticket so that we're in line again :)